app/src/main/java/com/onboarding/parkingsystemjava/mvp/model/ReserveModel.java

```diff
@@ -41,6 +43,7 @@ public void setEndDate(Calendar endDate) {
     }
 
     public ReserveComprobation reserveFieldsCheck(Reservation reservation) {
+        ReservationVerifier verifier = new ReservationVerifier(ParkingDatabase.getInstance());
```

Comment:
you shoud instantiate this on the presenter, and passit as a parameter for the model. 
this way you will can mockit when testing
check dependency inversion

Reply:
done